### ISSUE TITLE
Wired link mode when playing Legend of zelda four sword

### ISSUE THREAD

**Issue by ChrisTVH** [visualboyadvance-m/visualboyadvance-m#774]
![image](https://user-images.githubusercontent.com/58376852/98624484-ade4a080-22ec-11eb-8998-c206feaf2963.png)
We both use Windows 10 64 Bit
To connect the network we use hamachi
We tried in several ways but it does not work I will put them in situation,

The first computer pretending to be Player 1 acts as the server ...

The second computer that is the client pretends to be player two becomes client ...

The second computer that acts as a client when connecting
receives the Error that something has happened while the first computer marks it as if it had connected and I did not receive any error but it is not actually connected, This happens both as in the stable version of 2019 and in the version Monday, October 19 , 2020 11:01 PM 14484189 visualboyadvance-m-Win-64bit.zip, Eh seen that others have already had the same error and do not give any solution, which is wrong since they leave it halfway, also in case the ports connected are the same and apart from the emulator I have already played other games with hamachi connecting them locally through it that means that it is already a problem with the emulator as such, I would appreciate it if you would review this topic further, thank you for reading.

**Comment by ChrisTVH** [comment]
By the way, I forgot to say that I also tried to do it without having loaded the rom in the emulator and still the two computers are not synchronized with the connection

**Comment by rkitover** [comment]
Is it a problem with this game specifically? In general, GBA link does work.

**Comment by ChrisTVH** [comment]
@rkitover I only said to use that game to do the test but also detail that it is the emulator that does not connect the connection, it is this that gives the error, not the game

**Comment by rkitover** [comment]
@ChrisTVH yes I'm asking if you can try other games and see if they work with your configuration.

**Comment by ChrisTVH** [comment]
@rkitover  For you to understand what I mean, I have to send you photos
When the server is started
![image](https://user-images.githubusercontent.com/58376852/98824219-cd291e00-2411-11eb-81e1-e0c3e6ba0730.png)
Later when the client connects to the server
![image](https://user-images.githubusercontent.com/58376852/98825600-6efd3a80-2413-11eb-84eb-3c2c679ba6d4.png)

**Comment by rkitover** [comment]
Yes I understand that the game does not work, can you try other games?

**Comment by ChrisTVH** [comment]
Then I will try other games this time it will be with the pokemon
